pkg/utils/lookup/lookup.go

```diff
@@ -0,0 +1,26 @@
+package lookup
+
+import (
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+)
+
+// NodeIDsToLookupRequests ...
+func NodeIDsToLookupRequests(nodeIDs storj.NodeIDList) *pb.LookupRequests {
```

Comment:
Let's put these conversions into `pb` package. It seems to be about converting from and to pb.